addons/web_editor/static/src/js/editor/image_processing.js

```diff
@@ -390,9 +396,18 @@ async function _updateImageData(src, key = 'objectURL') {
     } else {
         value = URL.createObjectURL(blob);
     }
-    imageCache.set(src, Object.assign(currentImageData || {}, {[key]: value}));
+    imageCache.set(src, Object.assign(currentImageData || {}, {[key]: value, size: blob.size}));
     return value;
 }
+/**
+ * Returns the size of a cached image.
+ *
+ * @param {String} src used as a key on the image cache map.
+ * @returns {Number} size of the image in bytes.
+ */
+function _getImageSizeFromCache(src) {
+    return imageCache.get(src).size;
+}
```

Comment:
Not an issue at all but I'm a bit surprised that you created a function for this one line (quite explicit) instruction that is used only once. You can leave it like that if you prefer... I suppose it's a matter of taste

Reply:
The `imageCache` variable is not (and should not) be exposed to external files, therefore a published method is required for them to get such an information.